LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
Why not updating to 2.2.3 while we're on it? 2.2.1 wasn't the latest even 15 days ago :D